Podfile

```diff
@@ -8,7 +8,6 @@ target "Hakawai" do
 end
 
 target "HakawaiTests" do
-  pod 'Specta' 
-  pod 'Expecta'
+  pod 'Specta', :git => 'https://github.com/specta/specta.git', :branch => 'xcode_8'
+  pod 'Expecta', :git => 'https://github.com/specta/expecta.git'
```

Comment:
Is this change necessary?

Reply:
the master branch of Specta does not compile on xcode8. Additionally, when running pod install without pointing to the git for both specta and expecta, cocoapods can't find the libraries for some reason. Pointing directly fixes this.

Reply:
We should plan on deprecating these since they are removed from Cocoapods and no longer maintained.